main.go

```diff
@@ -1,25 +1,33 @@
 package main
 
 import (
-	"github.com/Albert221/medicinal-products-registry-api/api"
-	"github.com/graph-gophers/graphql-go/relay"
-	"log"
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/Albert221/medicinal-products-registry-api/api"
+	"github.com/graph-gophers/graphql-go/relay"
 )
 
 func main() {
 	downloader := Downloader{TargetFilename: "dataset.xml"}
 
 	updatedChan := make(chan bool)
-	go downloader.ScheduleDownloads(20 * time.Minute, updatedChan)
+	go downloader.ScheduleDownloads(20*time.Minute, updatedChan)
+
+	schema, err := api.NewSchema()
+	if err != nil {
+		panic(err)
```

Comment:
I think that logging the error is a better approach here, because error is not critical, application can always use the data it has downloaded before the fail and (theoretically) will update in the next try.